onadata/apps/logger/models/instance.py

```diff
@@ -441,15 +442,17 @@ def get_notes(self):
         # pylint: disable=no-member
         return [note.get_data() for note in self.notes.all()]
 
-    def get_review_status_and_comment(self):
+    def get_review_details(self):
         """
-        Return a tuple of review status and comment
+        Return a tuple of review status, comment and review date
         """
         try:
             # pylint: disable=no-member
-            status = self.reviews.latest('date_modified').status
-            comment = self.reviews.latest('date_modified').get_note_text()
-            return status, comment
+            review = self.reviews.latest('date_modified')
+            status = review.status
+            comment = review.get_note_text()
+            review_date = review.date_created
+            return status, comment, review_date
         except SubmissionReview.DoesNotExist:
             return None
```

Comment:
Seems the None portion is not being tested, it could be a source of potential bug, I would expect 3 None's to be returned.